Components/ReusableComponent/admin.js

```diff
@@ -0,0 +1,22 @@
+/* globals acf */
+import $ from 'jquery'
+
+acf.addAction('select2_init', function ($select, args, settings, field) {
+  const $fieldEl = field.$el
+
+  $select.on('change', { $fieldEl: $fieldEl }, function () {
```

Comment:
Doesn't this trigger on every select element in the backend? If so, it would be better to only fire on the specific select field of the reusable component.